README.md

```diff
@@ -3,7 +3,13 @@ The official [Angular](https://angular.io/) library for [Firebase](https://fireb
 
 <strong><pre>ng add @angular/fire</pre></strong>
 
-AngularFire smooths over the rough edges an Angular developer might encounter when implementing the framework-agnostic [Firebase JS SDK](https://github.com/firebase/firebase-js-sdk) & aims to provide a more natural developer experience by conforming to Angular conventions.
+_Note: If you want to `ng add` AngularFire and will be using Hosting/Cloud Functions then you will want to configure the 
```

Comment:
Hey @yharaskrik, just a suggestion. I think it would sound a bit better like:
_Note: If you want to `ng add` AngularFire and will be using Hosting/Cloud Functions, **you need to configure**...